src/services/auth.ts

```diff
@@ -14,16 +14,27 @@ const USERS_URL = new URL(urlWithEndSlash(process.env.REACT_APP_USERS_URL!), CLI
  * which does not contain the user data from the backend.
  */
 
+export enum loginState {
+  unauthenticated = "unauthenticated",
+  identified = "identified",
+  needs_registration = "needs-registration",
```

Comment:
The convention for the enum names is also PascalCase, i.e. `NeedsRegistration` instead of `needs_registration` etc.

Reply:
Done